app/interfaces/api/logger.rb

```diff
@@ -12,17 +12,24 @@ def after
     private
 
     def log_api_request(method, path, data)
-      log_api('api-request', method:, path:, data:)
+      log_api(:info,
+              'api-request',
+              { method:, path:, data: })
```

Comment:
Hi Alex, do the params need to be passed to the hash in the log_api method? Or does that not make any difference? 
 
 `{ method: method, path: path, data: data}`

Reply:
Hi Katy, this is known as Shorthand Hash Syntax. When you're declaring a Hash and you have a variable which is the same name as the key you don't need to explicitly write it down. `method:` by itself is the same thing as `method: method`

Reply:
Thanks @AlexRyanShep That is really helpful :)